documentation/book/proc-configuring-kafka-brokers.adoc

```diff
@@ -5,15 +5,18 @@
 [id='proc-configuring-kafka-brokers-{context}']
 = Configuring Kafka brokers
 
+You can configure an existing Kafka broker, or create a new Kafka broker with a specified configuration.
+
 .Prerequisites
 
-* An {ProductPlatformName} cluster
-* A running Cluster Operator
+* An {ProductPlatformName} cluster exists.
```

Comment:
_ exists_ is strange. It always exists somewhere. What about _is available_? Or _is running_?

Reply:
Changed to "is available".